WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -1812,7 +1816,19 @@ public static void openJetpackForDeeplink(@NonNull Context context, String actio
         context.startActivity(intent);
     }
 
-    public static void openPromoteWithBlaze(@NonNull Context context, PostModel postModel) {
-        Toast.makeText(context, R.string.button_promote_with_blaze, Toast.LENGTH_LONG).show();
+    public static void openPromoteWithBlaze(@NonNull Context context,
+                                            @Nullable PostModel postModel,
+                                            @NonNull BlazeFlowSource source) {
+        Intent intent = new Intent(context, BlazeParentActivity.class);
+        if (postModel != null) {
+            PostUIModel postUIModel =
+                    new PostUIModel(postModel.getRemotePostId(),
+                            postModel.getTitle(),
+                            postModel.getLink(),
+                            postModel.getFeaturedImageId());
```

Comment:
np:❓ Any reason you prefer this formatting instead of:
```java
PostUIModel postUIModel = new PostUIModel(
    postModel.getRemotePostId(),
    postModel.getTitle(),
    postModel.getLink(),
    postModel.getFeaturedImageId()
);
```

Reply:
Nope, other than my carelessness, nothing else. 😅 . Changed to the suggested format with c9e117bce1f8057e6ec2253ed345e8f628e7918d